Library/Homebrew/cmd/untap.rb

```diff
@@ -28,6 +31,20 @@ def untap
       tap = Tap.fetch(tapname)
       odie "Untapping #{tap} is not allowed" if tap.core_tap?
 
+      installed_tap_formulae = Formula.installed.select { |formula| formula.tap == tap }
+      installed_tap_casks = Cask::Caskroom.casks.select { |cask| cask.tap == tap }
+
+      if installed_tap_formulae.length.positive? || installed_tap_casks.length.positive?
+        if args.force?
+          opoo "Untapping #{tap} even though it contains formulae or casks that are currently installed."
```

Comment:
Would be nice if this listed the formulae and casks, maybe?

Reply:
Done